core/js/src/main/scala/org/log4s/log4sjs/Log4sLoggerFactory.scala

```diff
@@ -10,7 +10,7 @@ class Log4sLoggerFactory extends ILoggerFactory {
   private[this] final class Log4sLoggerInstance private[Log4sLoggerFactory] (private[this] val name: String) extends Logger {
     private[this] val path = LoggerParser(name)
 
-    override def getName = name
+    override def getName() = name
```

Comment:
All these parens are just fixing warnings, right? :+1: to them.

Reply:
OK, I'll fix it for both `scala-3.0.0-M1` and `scala-3.0.0-M2`. Yes, those parens are fixing warnings. It was extra annoying in Scala 3. 😅  I didn't fix infix warnings and Java converters warnings though.

Reply:
I usually like to be clean on warnings, but this project crosses so many versions, and already has so much duplication, I gave up here.  But these that can be fixed without hacks are wonderful.  Thanks!

Reply:
Yeah I understand. That's why I didn't handle the other two warning issues. 🙂